packages/usa-in-page-navigation/src/usa-in-page-navigation.twig

```diff
@@ -0,0 +1,31 @@
+{% macro inpagenav(items, base_class) %}
+
+  {% set list_item_class = 'usa-in-page-nav__item' %}
+  {% set current_class = 'usa-current' %}
+
+  <main id="main-content" class="main-content">
+    {% if heading %}
+      <h1>{{ heading }}</h1>
+    {% endif %}
+
+    {% if items %}
+      {% for item in items %}
+        {% if item.is_subheading %}
+          <h3>{{ item.section_heading }}</h3>
+        {% else %}
+          <h2>{{ item.section_heading }}</h2>
+        {% endif %}
+        <p>{{ item.content }}</p>
+      {% endfor %}
+    {% endif %}
+  </main>
+
+  <aside class="usa-in-page-nav"></aside>
```

Comment:
@mitchmoccia  @mejiaj 
Wondering about the benefits of nesting the `<nav>` inside an `<aside>`. I believe it is valid (depending on how it is defined), but feels semantically unnecessary and appears to just add an extra layer for users to tab through.

Can we just make this element the `<nav>` itself or switch to a `<div>` (if it needs a wrapper)?

Reply:
The [HTML5 standard mentions [html.spec.whatwg.org]](https://html.spec.whatwg.org/multipage/sections.html#the-aside-element:~:text=The%20element%20can%20be%20used%20for%20typographical%20effects%20like%20pull%20quotes%20or%20sidebars%2C%20for%20advertising%2C%20for%20groups%20of%20nav%20elements%2C%20and%20for%20other%20content%20that%20is%20considered%20separate%20from%20the%20main%20content%20of%20the%20page.):

>The element can be used for typographical effects like pull quotes or sidebars, for advertising, for groups of [nav](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) elements, and for other content that is considered separate from the main content of the page.

And they have an example below (third example down):

![image](https://user-images.githubusercontent.com/3385219/185478788-f6f4547d-574c-4617-a329-ad6a0ce87432.png)

**TL;DR** I can see both sides of the argument. Semantically I _think_ it's okay. The final decision depends on the screen reader experience. Unless others understand the use case better, please chime in!

Reply:
In the same spec, it does show this example for the nav element. https://html.spec.whatwg.org/multipage/sections.html#the-nav-element
> In the following example, there are two [nav](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) elements, one for primary navigation around the site, and one for secondary navigation around the page itself.

![image](https://user-images.githubusercontent.com/93996430/185481950-ac2fb483-d9fa-461c-89ed-14b878998744.png)

FWIW: I did poke around at other implementations and found that in its own implementation of a similar component (The "In this article" TOC on the right), Mozilla uses an `<aside>` with a nested `<section>` and no `<nav>`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside

Other popular design systems use `<nav>` with a nested list or just a `<div>` with a nested list. 

So my tl;dr is that it seems like there are a lot of possible correct paths here regarding if it is a nav or aside (or both), and that it might just depend on how we define "tangential" or "indirectly related" content. Users should still be able to find what they need with either. Change would only be needed here if we want to remove an extra layer when users navigate by element, but it shouldn't affect users who tab into it. 

[Reference:](https://html.spec.whatwg.org/multipage/sections.html#the-aside-element)
>The [aside](https://html.spec.whatwg.org/multipage/sections.html#the-aside-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) a section of a page that consists of content that is tangentially related to the content around the [aside](https://html.spec.whatwg.org/multipage/sections.html#the-aside-element) element, and which could be considered separate from that content. Such sections are often represented as sidebars in printed typography.

>The element can be used for typographical effects like pull quotes or sidebars, for advertising, for groups of [nav](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) elements, and for other content that is considered separate from the main content of the page.

Reply:
Love the tag team analysis!! I did some quick testing in VoiceOver and it doesn't seem to make a difference... I did make the decision to use aside > nav based on the w3c example but will go with whatever makes most sense.